src/js/components/DataTable/Header.js

```diff
@@ -139,7 +139,11 @@ const Header = forwardRef(
           )}
 
           {(selected || onSelect) && (
-            <TableCell background={background || cellProps.background}>
+            <TableCell
+              background={background || cellProps.background}
+              plain="fill"
```

Comment:
I find `plain="fill"`to be counter intuative.

Reply:
Valid feedback. The reason we cannot just say `plain={true}` is because `plain` as a boolean is already a documented property on TableCell. Unfortunately, its behavior does not align with general grommet stylings, where assigning `plain` would remove padding (in the TableCell case, padding is still applied when caller applies `plain`). For backwards compatibility, we needed to introduce either a new prop or a new value on `plain` that would indicate for the TableCell case that we want it to be trulyyy plain (no pad, the content should take up the entirety of the cell). 

Eric and I were thinking `fill` because in other areas of Grommet, `fill` means to take up all available space. Any other words that come to mind that mean a similar thing?

Reply:
Have updated to `noPad` to be more intuitive.